bgpd/bgp_vty.c

```diff
@@ -10398,6 +10397,42 @@ DEFUN (show_bgp_instance_all_summary,
   return CMD_SUCCESS;
 }
 
+DEFUN (show_bgp_ipv4_summary, 
+       show_bgp_ipv4_summary_cmd,
+       "show bgp ipv4 summary {json}",
+       SHOW_STR
+       BGP_STR
+       "Address family\n"
+       "Summary of BGP neighbor status\n")
+{
+  return bgp_show_summary_vty (vty, NULL, AFI_IP, SAFI_MAX, use_json(argc, argv));
+}
+
+DEFUN (show_bgp_instance_ipv4_summary,
+       show_bgp_instance_ipv4_summary_cmd,
+       "show bgp " BGP_INSTANCE_CMD " ipv4 summary {json}",
+       SHOW_STR
+       BGP_STR
+       BGP_INSTANCE_HELP_STR
+       "Address family\n"
+       "Summary of BGP neighbor status\n")
+{
+  return bgp_show_summary_vty (vty, argv[1], AFI_IP, SAFI_MAX, use_json(argc, argv));
+}
+
+DEFUN (show_bgp_instance_ipv4_all_summary,
+       show_bgp_instance_ipv4_all_summary_cmd,
+       "show bgp " BGP_INSTANCE_ALL_CMD " ipv4 summary {json}",
+       SHOW_STR
+       BGP_STR
+       BGP_INSTANCE_ALL_HELP_STR
+       "Address family\n"
+       "Summary of BGP neighbor status\n")
+{
+  return bgp_show_summary_vty (vty, argv[1], AFI_IP, SAFI_MAX, use_json(argc, argv));
+}
+
+#ifdef HAVE_IPV6
```

Comment:
Let's not add in any #ifdef HAVE_IPV6( it's preferable to just remove the #endif down below in the code